iwyu.cc

```diff
@@ -3596,6 +3596,11 @@ class IwyuAstConsumer
     ParseFunctionTemplates(context.getTranslationUnitDecl());
 
     TraverseDecl(context.getTranslationUnitDecl());
+
+    // Check if any unrecoverable errors have occurred
+    // For ex/ No sense continuing if an #include-ed file isn't found
```

Comment:
Unusual abbreviation there, `For ex/` -- how about just `There's no sense continuing [...]`

Reply:
Fixed in latest change.